submissions/asaMitaka/dom/style.css

```diff
@@ -0,0 +1,126 @@
+/* content */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    width: 100%;
+    height: 100%;
+}
+
+header {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    font-size: 30px;
+}
+
+.content {
+    display: flex;
+    justify-content: space-evenly;
+    height: 96vh; 
+}
+
+/* asideItem */
+
+.asideBlock {
+    width: 300px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+}
+
+.asideBtn {
+    width: 200px;
+    height: 50px;
+    border: 1px solid black;
+    background-color: gainsboro;
+    cursor: pointer;
+}
+
+.asideBtn:hover {
+    background-color: azure;
+    border: 1px solid blue;
+}
+
+/* mainItems */
+
+.mainBlock {
+    display: flex;
+    align-items: center;
+    min-width: 300px;
+    border: 1px solid black;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: gainsboro;
+}
+
+.mainHeader {
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    margin-top: 30px;
+    text-shadow: -2px 0px 10px rgba(150, 122, 129, 0.39);
+}
+
+.mainContent {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 30px;
+    align-items: center;
+}
+
+.mainImg {
+    /* border: 1px solid black; */
+    width: 200px;
+    height: 200px;
+}
+
+.mainDescr {
```

Comment:
Don't forget about mobile version
![image](https://user-images.githubusercontent.com/16196199/183964432-d0bf6d1b-d4e5-4c68-8943-de990c829f8e.png)

Reply:
It's the unaffordable luxury of making such large indents on mobile screens, try to reduce paddings for the mobile version
![image](https://user-images.githubusercontent.com/16196199/184121976-ffeabb64-982d-4a2f-80d2-00d3fbd5bdf8.png)